atomic_reactor/plugins/post_koji_import.py

```diff
@@ -682,6 +670,17 @@ def _update_extra(self, extra):
         if self.workflow.data.all_yum_repourls:
             extra['image']['yum_repourls'] = self.workflow.data.all_yum_repourls
 
+        if is_flatpak_build(self.workflow):
+            flatpak_util = FlatpakUtil(workflow_config=self.workflow.conf,
+                                       source_config=self.workflow.source.config,
+                                       composes=resolve_comp_result['composes'])
+            flatpak_compose_info = flatpak_util.get_flatpak_compose_info()
+            if flatpak_compose_info:
+                koji_metadata = flatpak_compose_info.koji_metadata()
+                koji_metadata['flatpak'] = True
```

Comment:
Why not set this directly inside the `koji_metadata()`?

Reply:
good idea :D. Updated